flink-table-store-format/src/main/java/org/apache/flink/table/store/format/orc/filter/OrcFilters.java

```diff
@@ -344,4 +344,27 @@ public String toString() {
             return "OR(" + Arrays.toString(preds) + ")";
         }
     }
+
+    /** An AND predicate that can be evaluated by the OrcInputFormat. */
+    public static class And extends Predicate {
+        private final Predicate[] preds;
+
+        public And(Predicate... predicates) {
```

Comment:
Adds the java doc of the constructor which style follows the `OR`, `NOT` etc:
```
/**
   * Creates an AND predicate.
   *
   * @param predicates The disjunctive predicates.
   */
```

Reply:
Thanks @SteNicholas Done